.github/workflows/build.yml

```diff
@@ -1,6 +1,10 @@
 name: Build and Test
 
-on: [pull_request]
+on: 
+  pull_request:
+  push:
+    branches:
+      - master
```

Comment:
Will this still support building PR branches?

Reply:
Yea the syntax here means build on every PR, and build on pushes to master (and pushes happen on merges)